src/urllib3/connectionpool.py

```diff
@@ -391,6 +390,19 @@ def _make_request(
             :class:`urllib3.util.Timeout`, which gives you more fine-grained
             control over your timeouts.
         """
+
+        # This value should already be assigned to the `url` parameter
+        if "request_url" in httplib_request_kw:
+            del httplib_request_kw["request_url"]
+
+        # The key above and all the following keys need removed because conn.request() will complain of
+        # unexpected keyword arugments
+        preload_content: bool = httplib_request_kw.pop("preload_content", True)
+        decode_content: bool = httplib_request_kw.pop("decode_content", True)
+        enforce_content_length: bool = httplib_request_kw.pop(
+            "enforce_content_length", True
+        )
```

Comment:
These values all had to be removed from `httplib_request` because of unexpected kwarg errors later when they were passed to `conn.request()`. I also needed to pass them on to `getresponse()`.

Reply:
As above, we can enumerate the parameters explicitly and remove any need for `**`

Reply:
Done